app.js

```diff
@@ -0,0 +1,142 @@
+const aws = require("aws-sdk");
+const s3 = new aws.S3({
+  apiVersion: "2006-03-01",
+});
+const saf = require("@mitre/saf");
+const fs = require("fs");
+const path = require("path");
+const winston = require("winston");
+const { createLogger, format, transports } = winston;
+let response;
+
+/**
+ *
+ * Event doc: https://docs.aws.amazon.com/apigateway/latest/developerguide/set-up-lambda-proxy-integrations.html#api-gateway-simple-proxy-for-lambda-input-format
+ * @param {Object} event - API Gateway Lambda Proxy Input Format
+ *
+ * Context doc: https://docs.aws.amazon.com/lambda/latest/dg/nodejs-prog-model-context.html
+ * @param {Object} context
+ *
+ * Return doc: https://docs.aws.amazon.com/apigateway/latest/developerguide/set-up-lambda-proxy-integrations.html
+ * @returns {Object} object - API Gateway Lambda Proxy Output Format
+ *
+ */
+
+exports.lambdaHandler = async (event, context) => {
+  const logger = createLogger({
+    level: process.env.LOG_LEVEL || "debug",
+    format: format.combine(format.timestamp(), format.simple()),
+    transports: [
+      new transports.Console({
+        format: format.combine(
+          format.timestamp({
+            format: "YYYY-MM-DDTHH:mm:ss.SSSZ",
+          }),
+          format.printf((info) => `${[info.timestamp]}\t${context.awsRequestId}\t${logger.level.toUpperCase()}\t${info.message}`)
+        ),
+      }),
+    ],
+  });
+
+  logger.log({
+    level: "debug",
+    message: "Logging Level set to  : " + logger.level.toUpperCase(),
+  });
+
+  if (!command_string) {
+    let command_string_message = "SAF CLI Command String argument is required. See http://saf-cli.mitre.org for more details.";
+    logger.info(command_string_message);
+    throw new Error(command_string_message);
+  }
+
+  if (CLI_COMMAND.trim() === "view" && CLI_FUNCTION.trim() === "heimdall") {
+    let view_heimdall_message = "You cannot use the 'saf view:heimdall' command in this environment.";
+    logger.info(view_heimdall_message);
+    throw new Error(view_heimdall_message);
+  }
+
+  // TODO: REMOVE ALL THESE COMMENTS
+  // TODO: Decide is we want to catch undefined saf-cli command groupings https://stackoverflow.com/questions/15201939/jquery-javascript-check-string-for-multiple-substringsa
+  // TODO: Removed hardcoded data and move to lambda params
+
+  const HEC_TOKEN = "473b3297-1d88-4740-96ff-e6048e51b785";
+  const SPLUNK_SERVER = "splk1.efficacy.online";
```

Comment:
Might want to remove this and clear git history
```suggestion
  const HEC_TOKEN = "YOUR_HEC_TOKEN";
  const SPLUNK_SERVER = "YOUR_SPLUNK_SERVER";
```